composer.json

```diff
@@ -18,7 +18,7 @@
     "repositories" : [
         {
             "type" : "git",
-            "url": "git@github.com:flash-global/connect-client.git"
+            "url": "https://github.com/flash-global/connect-client.git"
```

Comment:
Why did you changed ssh to https ?

Reply:
Travis Ci can't cloning public repository with ssh method. https link is needed

Reply:
https://travis-ci.org/flash-global/connect-package/builds/433995441?utm_source=github_status&utm_medium=notification